### ISSUE TITLE
Please help update test_runner_url configuration documentation

### ISSUE THREAD

**Issue by yanj-github** [cta-wave/dpctf-deploy#117]
OF configuration is changed and improved part of this issue:
https://github.com/cta-wave/device-observation-framework/issues/90
Can you help update test_runner_url configuration documentation please?
https://github.com/cta-wave/dpctf-deploy?tab=readme-ov-file#configure-the-observation-framework

test_runner_url now required to include web root e,g: http://localhost:8000/_wave/

**Comment by louaybassbouss** [comment]
@yanj-github do you have a new tag for the OF after the fix/change?

**Comment by louaybassbouss** [comment]
yes it is better to work for now in a feature branch and later do the tag when you merge with master. The deploy points anyway to a tag of the OF so if you even do the change directly in master, it will still work.

**Comment by yanj-github** [comment]
feature branch is created please use this up until we all ready for next release. Thanks

**Comment by yanj-github** [comment]
> @yanj-github we tested the new OF version (feature branch) and changed the `test_runner_url` to `http://server:port/_wave/` but we got this error:
> 
> ```
> Checking frame 0...
> ...
> Checking frame 7170...
> Checking frame 7180...
> Checking frame 7190...
> INFO     Mezzanine QR code areas not detected successfully.
> INFO     QR code area for full scan is set to left half of image for mezzanine QR code.
> INFO     QR code area for full scan is set to [0, 0, 960, 1080] for mezzanine QR code, and [1275, 306, 1702, 726] for test status QR code.
> INFO     Analysing '/usr/app/recordings/20241118_160054.mp4'...
> ERROR    Serious error is detected! Failed to get configuration file from test runner 404 system is terminating!
> ```
> 
> if we switch to the old OF version (OF master branch), the observation is working fine. Any idea?

Do your server connectible on http://server:port/_wave/ ?

**Comment by louaybassbouss** [comment]
> Do your server connectible on http://server:port/_wave/ ?
> 
> Can you try open a web browser on OF set up PC can try load http://server:port/tests.json also please make sure correct OF code is used.  

@yanj-github the TR is running on a server in our data center (trillian....) and it is working with the old TR. We added /_wave/ to the URL as you mentioned. 

http://server:port/tests.json is available, in our case it is https://trillian.fokus.fraunhofer.de/tests.json

**Comment by yanj-github** [comment]
> > Do your server connectible on http://server:port/_wave/ ?
> > Can you try open a web browser on OF set up PC can try load http://server:port/tests.json also please make sure correct OF code is used.
> 
> @yanj-github the TR is running on a server in our data center (trillian....) and it is working with the old TR. We added /_wave/ to the URL as you mentioned.
> 
> http://server:port/tests.json is available, in our case it is https://trillian.fokus.fraunhofer.de/tests.json (is a problem using https? the server redirect to https automatically)

Hi @louaybassbouss can you please run OF in debugging log enabled (with --log debug option) and it will allow us to see more detailed log to further diagnose. Also please double check you have correct OF version when it runs it should display version v2.0.2 at the beginning.

**Comment by louaybassbouss** [comment]
> Hi @louaybassbouss can you please run OF in debugging log enabled (with --log debug option) and it will allow us to see more detailed log to further diagnose. Also please double check you have correct OF version when it runs it should display version v2.0.2 at the beginning.

Thanks @yanj-github , @iljagav can you please do the proposed steps by @yanj-github .

**Comment by louaybassbouss** [comment]
> For your information:
> 
> * Check OF version it should say V2.0.2 from the log at the beginning:
> 
> `INFO Device Observation Framework **(V2.0.2)** analysis started! `
> 
> * When you run debug mode you will be able to see the full log when OF pulling configuration files:
>   e.g:
> 
> ```
> DEBUG    Starting new HTTPS connection (1): trillian.fokus.fraunhofer.de:443
> DEBUG    https://trillian.fokus.fraunhofer.de:443 "GET /test-config.json HTTP/1.1" 200 None
> DEBUG    Starting new HTTPS connection (1): trillian.fokus.fraunhofer.de:443
> DEBUG    https://trillian.fokus.fraunhofer.de:443 "GET /tests.json HTTP/1.1" 200 None
> ... ...
> ```
> 
> I have checked here and it works fine with the last OF code with config.ini test_runner_url = https://trillian.fokus.fraunhofer.de/_wave/

Thanks @yanj-github one more questions, are you running the OF with or without docker?

**Comment by yanj-github** [comment]
@louaybassbouss I normally runs it without docker. Good question it might be a networking issue on from docker. Check that if your server is accessible from inside docker please?

**Comment by iljagav** [comment]
> @louaybassbouss I normally runs it without docker. Good question it might be a networking issue on from docker. Check that if your server is accessible from inside docker please?

@yanj-github issue found and fixed (in one of the docker files, there was still `v2.0.1`). The OF inside docker is working fine and we can see that `v2.0.2` in the logs. We will send you the OF results summary by mail during the day.

**Comment by yanj-github** [comment]
@louaybassbouss  and @iljagav I think this is implemented and working can we close this issue?

**Comment by yanj-github** [comment]
@louaybassbouss and @iljagav Document still requires to be updated.

**Comment by louaybassbouss** [comment]
@yanj-github which documentation you mean? do you mean under section "[Use specific version](https://github.com/cta-wave/dpctf-deploy/tree/v2.1.0?tab=readme-ov-file#use-specific-version-1)"?

**Comment by yanj-github** [comment]
@louaybassbouss https://github.com/cta-wave/dpctf-deploy/issues/117#issue-2652506873
